lib/sm/client.rb

```diff
@@ -110,6 +111,8 @@ def connection
         conn.use :breakers
         conn.request :multipart
         conn.request :json
+        # conn.request :curl, ::Logger.new(STDOUT), :warn
```

Comment:
I output to STDOUT because I just run my spec and it gives me the output that I need to send to MHV.